netbox/dcim/views.py

```diff
@@ -124,7 +124,7 @@ def post(self, request):
 
             if form.is_valid():
 
-                with transaction.atomic():
+                with transaction.atomic(using=router.db_for_write(self.queryset.model)):
```

Comment:
I think we should pass the Cable model here, as those are what's being deleted within the transaction.
```suggestion
                with transaction.atomic(using=router.db_for_write(Cable)):
```